config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
@milenaveneziani and @pwolfram I don't need the movingAverage.  I didn't see the reason to retain.  The nino3.4 algorithm specifies a 5-month average so it seemed unnecessary and unwise to have the possibility for the user to pick something else.

@milenaveneziani I don't understand the second comment about regionIndex.  Are you saying to refer to regionIndicesToPlot as before?  I can change that.